Bottom: 7dc5e5bcab773a09880cf99fc2a43a97da9827d4
Top:    db1c09bf1ee5cf0b3d74ced0894853e55d09bfe2
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-03 14:47:10 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/daemon/indirect.c b/daemon/indirect.c
index 2dba9bf..61f2601 100644
--- a/daemon/indirect.c
+++ b/daemon/indirect.c
@@ -155,6 +155,13 @@ static int do_mount_autofs_indirect(struct autofs_point *ap, const char *root)
 		goto out_rmdir;
 	}
 
+	ret = mount(NULL, root, NULL, MS_UNBINDABLE|MS_REC, NULL);
+	if (ret) {
+		warn(ap->logopt,
+		     "failed to set autofs mount unbindable path %s at %s",
+		     ap->path, root);
+	}
+
 	free(options);
 	options = NULL;
